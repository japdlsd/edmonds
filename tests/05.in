10 20
1 3 42
1 4 16
1 5 32
1 9 38
2 4 49
2 6 8
2 7 29
3 5 15
3 7 49
3 8 30
3 9 25
3 10 11
4 7 25
4 9 36
5 9 45
6 7 32
6 8 37
6 9 2
6 10 26
7 10 41