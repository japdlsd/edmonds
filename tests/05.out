138
6 8
5 9
3 10
2 7
1 4
